Podfile.lock

```diff
@@ -13,6 +13,6 @@ SPEC REPOS:
 SPEC CHECKSUMS:
   MSAL: 74bf430ec0f46dd4c79dde2596e2b553610ccab1
 
-PODFILE CHECKSUM: a5fd21670ef5dd405bcb0d5adf5591f8d31ee53f
+PODFILE CHECKSUM: e70aefde8398e9dc7b42edaa007f940c8a787366
 
-COCOAPODS: 1.9.1
```

Comment:
Might want to just update your Cocoapods rather than committing this.

Reply:
Fixed.